ray-operator/config/samples/pytorch-text-classifier/ray-job.pytorch-distributed-training.yaml

```diff
@@ -0,0 +1,59 @@
+# This RayJob is based on the "Fine-tune a PyTorch Lightning Text Classifier with Ray Data" example in the Ray documentation.
+# See https://docs.ray.io/en/master/train/examples/lightning/lightning_cola_advanced.html for more details.
+apiVersion: ray.io/v1
+kind: RayJob
+metadata:
+  generateName: pytorch-text-classifier-
+spec:
+  entrypoint: python /home/ray/kuberay/kuberay-master/ray-operator/config/samples/pytorch-text-classifier/fine-tune-pytorch-text-classifier.py
+  runtimeEnvYAML: |
+    pip:
+      - numpy
+      - datasets
+      - transformers>=4.19.1
+      - pytorch-lightning==1.6.5
+  rayClusterSpec:
+    rayVersion: '2.9.0'
+    headGroupSpec:
+      rayStartParams:
+        dashboard-host: '0.0.0.0'
+      template:
+        spec:
+          initContainers:
```

Comment:
Use `working_dir` instead of an init container. See [ray-service.stable-diffusion.yaml](https://github.com/ray-project/kuberay/blob/03f1a2e6939791abe896788a59f8171b6ad0b02b/ray-operator/config/samples/ray-service.stable-diffusion.yaml#L11) as an example.

Reply:
Ah nice, I didn't know you can reference remote zip files with working_dir

Reply:
done!